setup.py

```diff
@@ -200,7 +200,10 @@ def run(self):
     },
     'entry_points': {
         'console_scripts': [
+            'kiwi=kiwi:main',
+            'kiwi-ng=kiwi:main',
```

Comment:
I'd actually rather retire the `kiwi-ng` name... Do we really need it anymore?

Reply:
I'd say we don't but there might be people around using it. That's why I am also keeping the `kiwi-ng-3`. Specially if there is people using kiwi from pip.

But I agree that at certain point we should drop everything else than `kiwi`. This includes removing the compat code.

I think that perl based kiwi support fully ends on 2022 (together with SLE12), but I'd say we probably should drop that compat support on upstream before than that and also reduce the console_scripts.

Reply:
This is probably a good time to drop it as any...

Reply:
We are referring to kiwi-ng or kiwi-ng-3 in many places in the docs.
I'd say this needs to be addressed first. Also we need to ensure kiwi-ng is not used in OBS and probably there is other people scripting around kiwi out there.

Anyway I see your point. Probably we could announce it and then start with a first step having only `kiwi-ng`, `kiwi` and `kiwi-compat` and later in a later stage leave only `kiwi`.

In any case docs goes first/together and at least this should be addressed in this PR.

@schaefi do you have any thoughts

Reply:
I agree to the plan to get rid of it. But two important points were already mentioned:

- docs
- obs

I'm pretty sure if we deleted the kiwi-ng we will break all builds in obs. A patch in the build script is most likely needed. This however will be difficult because obs also supports building with the legacy kiwi version. afaik I coded the script in a way that it uses kiwi (for legacy builds) and kiwi-ng for next generation builds. If we now delete kiwi-ng we might open a can of worms that are hard to control.

So for now please let us keep both names and focus first how and if we can come up with a good solution on the obs side

Thoughts ?

Reply:
Just for the notes I believe that OBS is not calling `kiwi-ng` at all, at least it is not doing so in `obs-build/build-recipe-kiwi` and I doubt OBS calls KIWI anywhere else. Anyway I agree we can't do such a change without a proper coordination with the OBS folks and without proper announcements before actually removing any console script.

I lets update docs and align pip installation with the docs.

Reply:
I remember I wrote a function there that distinguishes by path, here is the magic from build-recipe-kiwi

```bash
    local kiwi_path=/usr/bin/kiwi kiwi_legacy=false
    if ! test -L "$BUILD_ROOT/usr/bin/kiwi" ; then
    kiwi_path=/usr/sbin/kiwi
        kiwi_legacy=true
    fi
```

I hope all this still works after the change, I think it should

Reply:
@davidcassany please double check before merge

Reply:
Thanks for the hint, I'll check this portion in some home project of mine in OBS and verify the recipe does not face any regression. I'll do the merge once confirmed it works as expected.

Reply:
You were right portion of the code in build-recipe-kiwi is causing issues against the suggestion I did. This part expects `/usr/bin/kiwi` to be a symlink, which was not the case with my suggestion.

I had to revert some of the changes, so pip will install `kiwi-ng` and keep `kiwi` and `kiwi-ng-3` as compat links at spec level.